packages/farcaster-js-neynar/scripts/generateSigner.ts

```diff
@@ -1,54 +1,33 @@
 /* eslint-disable eslint-comments/disable-enable-pair */
 /* eslint-disable no-console */
 /* eslint-disable @typescript-eslint/restrict-template-expressions */
-import { NeynarAPIClient, generateSignature } from '../src/index.js';
+import { NeynarAPIClient, waitForNeynarSignerApproval } from '../src/index.js';
```

Comment:
Do we still need this script now that the CLI exists?

Reply:
nope, between the cli and examples this is covered